tests/fourier/test_qnode_spectrum.py

```diff
@@ -32,6 +32,7 @@ def circuit_0(a):
 def circuit_1(a, b):
     qml.RZ(-a / 3, wires=0)
     qml.RX(a / 5, wires=1)
+    qml.CRZ(0.3, wires=[1, 0])
```

Comment:
Ah! So this is why it wasn't being picked up by the tests?

Reply:
Yes, I didn't think of adding in constant-parametrized gates in the examples - or at least not such that they had different spectra :man_facepalming: